src/js/lattice/lattice_types.ts

```diff
@@ -37,21 +27,23 @@ export enum LatticeTypeExtended {
     TRI_1b = "TRI_1b",
 }
 
-export type Vector = ArrayOf3NumberElementsSchema;
-
-export interface VectorsAsArray extends Array<Vector> {
-    0: Vector;
-    1: Vector;
-    2: Vector;
-}
-
-interface LatticeTypeConfig {
+export interface LatticeTypeConfig {
     label: string;
-    code: LatticeTypeSchema;
+    code: LatticeTypeEnum;
     editables: string[];
     editablesConventional: string[];
 }
 
+export const DEFAULT_LATTICE_UNITS = {
```

Comment:
We should try generating enums (not types as enums) in ESSE and reuse from there. Maybe we can tweak some options in the library used to convert json schemas to types.